proto/v1/v1/org_policy_service.proto

```diff
@@ -293,14 +291,21 @@ message DisableCopyDataPolicy {
   bool active = 1;
 }
 
-message ExportDataPolicy {
-  bool disable = 1;
-}
-
 // QueryDataPolicy is the policy configuration for querying data.
 message QueryDataPolicy {
   // The query timeout duration.
   google.protobuf.Duration timeout = 1;
+
+  // Disable export data in the SQL editor
+  bool disable_export = 2;
+
+  // The size limit in bytes.
+  // The default value is 100MB, we will use the default value if the setting not exists, or the limit <= 0.
+  int64 maximum_result_size = 3;
+
+  // The return rows limit.
+  // The default value is -1, means no limit.
```

Comment:
Not sure if it is too late, but I would like to make it consistent with the other flag that value <= 0 means unlimited. But not necessarily to be in this PR.